packages/wonder-blocks-dropdown/src/components/select-opener.tsx

```diff
@@ -231,99 +235,54 @@ const styles = StyleSheet.create({
     },
 
     caret: {
-        minWidth: 16,
+        minWidth: sizing.size_160,
     },
-});
-
-const stateStyles: Record<string, any> = {};
-
-const _generateStyles = (placeholder: boolean, error: boolean) => {
-    // "hash" the parameters
-    const styleKey = `${placeholder}-${error}`;
-    if (stateStyles[styleKey]) {
-        return stateStyles[styleKey];
-    }
-
-    // The color is based on the action color.
-    const actionType = error ? "destructive" : "progressive";
-    // NOTE: We are using the secondary action type for all the non-resting
-    // states as the opener is a bit different from a regular button in its
-    // resting/default state.
-    // TODO(WB-2007): Adopt design specs
-    const action = semanticColor.action.secondary[actionType];
-
-    const hoverStyling = {
-        borderColor: action.hover.border,
-        boxShadow: `inset 0 0 0 ${border.width.thin} ${action.hover.border}`,
-    };
-    const pressStyling = {
-        background: action.press.background,
-        color: placeholder
-            ? error
-                ? semanticColor.input.default.placeholder
-                : semanticColor.core.foreground.instructive.default
-            : semanticColor.input.default.foreground,
-        borderColor: action.press.border,
-        boxShadow: `inset 0 0 0 ${border.width.thin} ${action.press.border}`,
-        borderRadius: theme.opener.border.radius.press,
-    };
-
-    const currentState = error
-        ? semanticColor.input.error
-        : semanticColor.input.default;
-
-    const disabledStatesStyles = {
+    /**
+     * Theming
+     */
+    default: {
+        background: semanticColor.input.default.background,
+        border: `${border.width.thin} solid ${semanticColor.input.default.border}`,
+        color: semanticColor.input.default.foreground,
+        "::placeholder": {
```

Comment:
Since the select opener is a button instead of an `input` element, the `::placeholder` styles don't get applied! 

For example, in the disabled state, the placeholder is still using input placeholder default token, instead of the disabled token 
<img width="947" height="819" alt="Image" src="https://github.com/user-attachments/assets/e0c5c6b6-1be9-4146-ba42-0801774599e4" />

They're the same value anyways but thought I'd mention it! Applies to other places in the file that uses `::placeholder`!

Reply:
ohh good point! let me fix it.